src/Suave/Logging/TraceHeader.fs

```diff
@@ -9,45 +9,38 @@ open Suave.Utils.RandomExtensions
 /// of possible values, so you can be fairly certain a given request
 /// id is unique, given a good random number generator.
 type TraceHeader =
-  { /// if this is the 'first' traced request, then trace_id equals
+  { /// If this is the 'first' traced request, then trace_id equals
     /// req_id. If it's the second, then trace_id = req_parent_id
     /// or otherwise third or later then trace_id, req_id and req_parent_id
     /// are all disjunct
-    trace_id      : uint64
-    /// the request id assigned when suave received the http request
+    traceId      : uint64
+
+    /// The request id assigned when suave received the http request
     /// In ZipKin/Dapper-speak, this is the span id
-    req_id        : uint64
+    reqId        : uint64
+
     /// possibly a parent
     /// In ZipKin/Dapper-speak, this is the span parent id
-    req_parent_id : uint64 option }
+    reqParentId : uint64 option }
+
+  static member traceId_     = Property (fun x -> x.traceId) (fun v x -> { x with traceId = v })
```

Comment:
Hope nobody minds me dropping a comment in here! This is quite an interesting one for me, as I've struggled to decide on a consistent approach to structuring lens properties/functions. At first glance static members seemed ideal, as the lens itself from Type -> Property of Type is intrinsically bound to that type. 

However, when you start to define composite lenses, where should they live? They're no longer bound to a single type. How about lenses which aren't bound to a type in the first place? (For example, lenses for the fst and snd items of a 2-tuple). You also can't extend a type alias, so when you wish to define lenses on a specialisation of a generic type, that's not possible in the same way.

It's a tricky one, and I've ended up with a mix throughout the code I write using lenses. I'm still not really delighted by that as a general rule, but if anything i'm leaning back towards lenses always being effectively a naked function (or function pair, in this formulation). Slightly off topic perhaps, but as they're being commented on here, seemed a chance to see if others had strong opinions...